### PR TITLE
Resolved the issues Li mentioned in PR #9

### PR DESCRIPTION
**Pull request by ArabellaJi**
Resolved the issues Li mentioned in PR #9.
Fixes https://github.com/1thing-org/hatecrimetracker-app/issues/259.

### CODE REVIEW

File: firestore/incidents.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Incident(mdl.Model):
     title = mdl.TextField(required=True)
     title_translate = mdl.MapField(required=False)
     publish_status = mdl.MapField(
-        required=True, default={"twitter": None, "linkedin": None}
+        required=True, default={"twitter": None, "linkedin": None, "notification": None}
```
Comment (lima-1thing) [inline firestore/incidents.py:22]: why remove "notification"?

File: firestore/user_reports.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from firestore.cachemanager import INCIDENT_CACHE, INCIDENT_STATS_CACHE, flush_cache
+
+
+class UserReport(mdl.Model):
+    user_report_time = mdl.DateTime(required=True)
+    created_on = mdl.DateTime(auto=True)
+    user_report_location = mdl.TextField()
+    description = mdl.TextField(required=True)
+    description_translate = mdl.MapField(required=False)
+    attachments = mdl.MapField(required=False)
```
Comment (lima-1thing) [inline firestore/user_reports.py:16]: whi attachement is MapField?

File: firestore/user_reports.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+
+import dateparser
+from cachetools import cached
+from fireo import models as mdl
+
+from firestore.cachemanager import INCIDENT_CACHE, INCIDENT_STATS_CACHE, flush_cache
+
+
+class UserReport(mdl.Model):
```
Comment (lima-1thing) [inline firestore/user_reports.py:10]: should we have a status field for user report? all user reports must be reviewed/approved/rejected by admins.

File: main.py [hunk 4]
```diff
@@ -117,6 +118,21 @@ def get_incidents():
     }
 
 
+@app.route("/user_reports")
+def get_user_reports():
```
Comment (lima-1thing) [inline main.py:122]: who will call this RPC? Admins or end users? For end users, we should only load approved reports.

**Comment by ArabellaJi** [comment]
The destination branch of this PR is wrong. The changes should be merged into self-report-dev instead of main, so I opened a new PR and will close this one. The new PR is https://github.com/1thing-org/hatecrimetracker/pull/16.

File: firestore/user_reports.py (hunk 2 above)
Comment (ArabellaJi) [reply firestore/user_reports.py:16]: Resolved in https://github.com/1thing-org/hatecrimetracker/pull/16

File: firestore/user_reports.py (hunk 3 above)
Comment (ArabellaJi) [reply firestore/user_reports.py:10]: Resolved in https://github.com/1thing-org/hatecrimetracker/pull/16

File: main.py (hunk 4 above)
Comment (ArabellaJi) [reply main.py:122]: This problem is too complicated to solve in this PR. I will create a new issue for it.

**Pull request closed without merging**